src/components/Todo.jsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import plus from "../assets/plus.svg";
+import Task from "./Task";
+import { nanoid } from "nanoid";
+
+export const Todo = () => {
+  const [text, setText] = useState({ taskName: "" });
+  const [tasks, setTasks] = useState([]);
+
+  const handleText = (e) => {
+    const { name, value } = e.target;
+    setText((prev) => {
+      return {
+        ...prev,
+        [name]: value,
+      };
+    });
```

Comment:
As a suggestion, you could a normal string for the state as you only have one input and it easier to handle a primitive variable when setting state.

Reply:
got it, thanks